src/visualizers/ModFill.ts

```diff
@@ -39,8 +35,9 @@ modulus to consider.
 } as const
```

Comment:
Do you want me to change this and remove the hard-coded index 1000 limit as part of this PR, or should we just file both as an issue, since they are not really germane to this PR?

Reply:
Whatever work flow seems best but I'd like to have both fixed eventually.

Reply:
This PR is pretty huge already, so I'm filling an issue for both ModFill items and then will resolve this conversation.